sdk/servicebus/test-resources.json

```diff
@@ -3,25 +3,7 @@
   "contentVersion": "1.0.0.0",
   "parameters": {
     "baseName": {
-      "type": "string",
-      "defaultValue": "[resourceGroup().name]",
-      "metadata": {
-        "description": "The base resource name."
-      }
-    },
-    "subscriptionId": {
-      "type": "string",
-      "defaultValue": "[subscription().subscriptionId]",
-      "metadata": {
-        "description": "The subscription ID to which the application and resources belong."
-      }
-    },
-    "tenantId": {
-      "type": "string",
-      "defaultValue": "[subscription().tenantId]",
-      "metadata": {
-        "description": "The tenant ID to which the application and resources belong."
```

Comment:
Does removing these params remove any necessary information used for samples, etc.? (probably not, but just checking :P)